### DISCUSSION TITLE
OCR “自有”“有道”成功使用，但识别结果不理想

### DISCUSSION THREAD

**Discussion by Seryta** [shuax/MouseInc.Settings#385] [Show and tell]
根据作者的说明，“自用”无法使用应该是TLS1.3未启用的原因，经过摸索之后找到了解决方案：

1. win+s 搜索 Internet选项，点击“高级”，勾选“SSL3.0" "TLS1.3"
![image](https://user-images.githubusercontent.com/31036794/189018117-9555c2c7-9882-4833-9ac8-6f8201ce9d6d.png)
2. 上面配置完重启之后并没有效果，于是参考[此链接](https://blog.csdn.net/ximenjianxue/article/details/125958000)，修改注册表
```
#查询
reg query "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client"
#添加
reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client" /v "Enabled" /t REG_DWORD /d "1" /f

reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client" /v "DisabledByDefault" /t REG_DWORD /d "0" /f

#添加Server
reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server" /v "Enabled" /t REG_DWORD /d "1" /f

reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server" /v "DisabledByDefault" /t REG_DWORD /d "0" /f
```
3. 重启系统，这时候”自用“OCR正常可用 
> 注意：我将 ocr.shuax.com 直连，而没有走代理

但是发现这里有点问题，就是OCR的识别效果似乎下降了，这个不确定

识别文本：
> C:\Users\D\Desktop\3\新建文件夹\DwnlData
”自有“识别结果：
> CUsersDDesktop3新建文件DwnDat
”有道“识别结果：
> C:UserslDVDesktop(3(新建文件)DwnlData
"飞桨”识别结果：
> C:\Users\D\Desktop)3)新建文件夹\DwnlData

顺带一提，“自有”和“有道”有时候不稳定，可能需要两次才能成功

**Comment by shuax** [comment]
不需要tls1.3了，我把1.2开上了，之前是我nginx编译有问题。

**Reply by kevzcr** [reply]
将tls１.３关掉了，`自用`可用（有道依然不能用），但掉字特别严重，以前从未有过这种情况。

![image](https://user-images.githubusercontent.com/88956040/189467046-873d7587-5c51-4318-803b-db68702ebadd.png)

**Comment by shuax** [comment]
自有改了一个模型，好像识别率要好一点了。
